api/v1/utils/time.js

```diff
@@ -0,0 +1,7 @@
+/* jshint esversion: 6 */
+
+const MILLISECONDS_PER_SECOND = 1000;
```

Comment:
Yeah. The only weird thing about a `utils/const.js` is that the config has to import a v1 utility. Maybe we could just have a global `const.js`.